source/agora/consensus/Fee.d

```diff
@@ -525,10 +524,11 @@ public class FeeManager
         auto utxoset = new TestUTXOSet();
 
         Amount double_stake = Amount.MinFreezeAmount; assert(double_stake.mul(2));
+        Amount stake_with_excess = Amount.MinFreezeAmount; assert(double_stake.add(Amount(19))); // this should not change distribution
```

Comment:
Is the `assert` be on `double_stake` or `stake_with_excess` ?

Reply:
In any case, you can use the operators overloads here if you want, since it's not testing the addition / multiplication themselves.